### PR TITLE
Android fixes

### PR DESCRIPTION
**Pull request by evelikov**
Bunch of Android fixes while browsing through. Another stable release worthy material.

### CODE REVIEW

**Review by dvrogozh — APPROVED** [review]
This does look good to me. However, are these blind changes or you had a chance to verify build and maybe test?

**Comment by evelikov** [comment]
Build tested with hacked up the meson build.

File: va/android/va_android.cpp [hunk 1]
```diff
@@ -28,14 +28,12 @@
...
 #include "va_internal.h"
 #include "va_trace.h"
-#include "va_fool.h"
 #include "va_android.h"
 #include "va_drmcommon.h"
 #include "va_drm_utils.h"
 #include <stdarg.h>
 #include <unistd.h>
 #include <sys/types.h>
-#include <sys/stat.h>
```
Comment (dvrogozh) [inline va/android/va_android.cpp:38]: Drop this include is the only thing which bother me a little. For example, https://manpages.ubuntu.com/manpages/xenial/man2/open.2.html:
```
       #include <sys/types.h>
       #include <sys/stat.h>
       #include <fcntl.h>
```

Though there are other mans over network which just list `#include <fcntl.h>`. Maybe safer to leave this include here? I am not sure how android includes would be structured.

Comment (evelikov) [reply va/android/va_android.cpp]: AFAICT there is lots of magic about these includes - both C runtime (glibc vs bionic) and version specific. So it's safer to keep it as-is. Pushed an update with the include preserved.

**Review by dvrogozh — APPROVED** [review]
looks good

**Review by xmao4-intel — APPROVED** [review]

**Review by TianmiChen — APPROVED** [review]

**Comment by evelikov** [comment]
Happy one month anniversary \o/

Thanks for merging.

**Pull request merged**